rustls/src/server/server_conn.rs

```diff
@@ -435,6 +437,9 @@ pub struct ServerConfig {
     ///
     /// [RFC8779]: https://datatracker.ietf.org/doc/rfc8879/
     pub cert_decompressors: Vec<&'static dyn compress::CertDecompressor>,
+
+    /// Determines how invalid SNI in ClientHello is handled.
```

Comment:
Editorial suggestion:

```suggestion
    /// Policy for how an invalid Server Name Indication (SNI) value from a client is handled.
```